packages/contracts/test/plugins/governance/majority-voting/addresslist/addresslist-voting.ts

```diff
@@ -135,6 +136,79 @@ describe('AddresslistVoting', function () {
     });
   });
 
+  describe('plugin interface: ', async () => {
+    it('does not support the empty interface', async () => {
+      expect(await voting.supportsInterface('0x00000000')).to.be.eq(false);
+    });
+
+    it('supports the `IERC165Upgradeable` interface', async () => {
+      // @ts-ignore
+      const IERC165Upgradeable = await hre.artifacts.readArtifact(
+        'IERC165Upgradeable'
+      );
+      const iface = new ethers.utils.Interface(IERC165Upgradeable.abi);
+
+      expect(await voting.supportsInterface(getInterfaceID(iface))).to.be.eq(
+        true
+      );
+    });
+
+    it('supports the `IPlugin` interface', async () => {
+      // @ts-ignore
+      const IPlugin = await hre.artifacts.readArtifact('IPlugin');
+      const iface = new ethers.utils.Interface(IPlugin.abi);
+
+      expect(await voting.supportsInterface(getInterfaceID(iface))).to.be.eq(
+        true
+      );
+    });
+
+    it('supports the `IProposal` interface', async () => {
+      // @ts-ignore
+      const IProposal = await hre.artifacts.readArtifact('IProposal');
+      const iface = new ethers.utils.Interface(IProposal.abi);
+
+      expect(await voting.supportsInterface(getInterfaceID(iface))).to.be.eq(
+        true
+      );
+    });
+
+    it('supports the `IMajorityVoting` interface', async () => {
+      // @ts-ignore
+      const IMajorityVoting = await hre.artifacts.readArtifact(
+        'IMajorityVoting'
+      );
+      const iface = new ethers.utils.Interface(IMajorityVoting.abi);
+
+      expect(await voting.supportsInterface(getInterfaceID(iface))).to.be.eq(
+        true
+      );
+    });
+
+    it('supports the `IMajorityVoting` interface', async () => {
```

Comment:
duplicated

Reply:
Fixed in ed5e30b361ad3d3d084527599af1cc3935cf9021